packages/featureflags/src/vars/index.js

```diff
@@ -0,0 +1 @@
+export const psds2020Colors = 'psds2020Colors';
```

Comment:
Not sure how I feel about specifying these in this package.  Exporting it becomes a part of the public api.  But it's temporary, so when we remove, that's a breaking change.  But it's nice, because it's a single shared place where specify the name of this flag.  The alternative might be to put this string into each of the 3 first wave components instead.  What do you think?  What seems best?

Reply:
I'll hardocde those I think based on our 1:1 discussion